resources/views/companies/joboffers/edit.blade.php

```diff
@@ -42,8 +42,17 @@
                                    placeholder="Chief Executive Officer" value="{{ $joboffer->title or '' }}" required>
                         </div>
 
-
                         <div class="form-group">
+                            <label>Offer information type</label>
+                            <select class="form-control information_type_selector">
+                                <option value="" @if(!$joboffer || ($joboffer->description == '' && $joboffer->redirect_url == '')) selected @endif disabled>Select a type...
```

Comment:
You check the description and redirect_url for empty strings, but their default/disabled values are null.